crossfire/__init__.py

```diff
@@ -57,12 +58,11 @@ def occurrences(
     )
 
 
-def _make_flatten(data, keys):
-    for item in data:
-        for key in keys:
-            item.update({f"{key}_{k}": v for k, v in item.get(key).items()})
-            item.pop(key)
```

Comment:
As I am not converting anymore to dict, I put this code back in the `flatten` function when dealing with dict...